README.md

```diff
@@ -106,6 +106,58 @@ jobs:
         password: ${{ secrets.password }}
 ```
 
+## Laravel
+#### Build and Publish Laravel Website
+```yml
+# This is a basic workflow to help you get started with Actions
+
+name: 🚀 Deploye
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+    types: [closed]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  web-deploy:
+    name: 🎉 Deploying
+    runs-on: ubuntu-latest
+    steps:
+    - name: 🚚 Get latest code
+      uses: actions/checkout@v3
```

Comment:
I think actions/checkout@v4 should be used because it is more up to date and has been tested as working